config/vufind/BrowZine.ini

```diff
@@ -67,6 +72,20 @@ browzineWebLink = "View Complete Issue|browzine-issue|https://assets.thirdiron.c
 fullTextFile = "PDF Full Text|browzine-pdf|https://assets.thirdiron.com/images/integrations/browzine-pdf-download-icon.svg"
 retractionNoticeUrl = "View Retraction Notice|browzine-retraction"
 
+; The bestIntegrationLink service is not yet documented on that wiki page, but it provides
+; "the best link available" according to Third Iron's algorithm. See
+; https://support.thirdiron.com/support/solutions/articles/72000569332-open-access-linking-and-configuration-options).
+; The label and icon values specified here are defaults, but are overridden based on the
+; type of "best" link actually provided for a given title. See [BestIntegratorLinks] below. 
+;bestIntegratorLink = "View Online|browzine-best"
+
+; When 'bestIntegratorLink' is enabled in DOIServices, this section defines what label and
+; icon to display based on the type of "best" link.
+; See also useBrowzineLabel within [IdentifierLinks] above.
+[BestIntegratorLinks]
+fullTextFile = "PDF Full Text|browzine-pdf|https://assets.thirdiron.com/images/integrations/browzine-pdf-download-icon.svg"
+retractionNoticeUrl = "View Retraction Notice|browzine-retraction"
```

Comment:
This data is 100% copy/paste from [DOIServices] above.  But I imagine the actual use case is that someone comments out `fullTextFile` and `retractionNoticeUrl` under [DOIServices], using bestIntegratorLink instead.  So I still need to get that label and icon config somewhere.